memphis/memphis.py

```diff
@@ -180,14 +183,47 @@ async def producer(self, station_name, producer_name, generate_random_suffix=Fal
                 createProducerReq, indent=2).encode('utf-8')
             err_msg = await self.broker_manager.request("$memphis_producer_creations", create_producer_req_bytes)
             err_msg = err_msg.data.decode("utf-8")
-
-            if err_msg != "":
+            err_msg = json.loads(err_msg)
+            if err_msg['error'] != "":
                 raise Exception(err_msg)
+            
+            self.t_schema_updates = await self.start_listen_for_schema_updates(station_name)
```

Comment:
this is not good this way you always have 1 subscription per the entire connection, what happens if I'm creating a producer to station x, then creating a producer to station y, subscription fo station y will delete the subscription for x.
You have to maintain a map of subscriptions as well as producers per station name.
Please take a look on the JS implementation